generation/um/wslapi/generate.rsp

```diff
@@ -0,0 +1,22 @@
+@../../settings.rsp
+@../../remap.rsp
+--exclude
+operator|(WSL_DISTRIBUTION_FLAGS, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS
+operator|=(WSL_DISTRIBUTION_FLAGS &, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS &
+operator&(WSL_DISTRIBUTION_FLAGS, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS
+operator&=(WSL_DISTRIBUTION_FLAGS &, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS &
+operator~(WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS
+operator^(WSL_DISTRIBUTION_FLAGS, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS
+operator^=(WSL_DISTRIBUTION_FLAGS &, WSL_DISTRIBUTION_FLAGS):WSL_DISTRIBUTION_FLAGS &
```

Comment:
This should be fine to remove now, `exclude-enum-operators` gets it automatically
```suggestion
```